pkg/artifactory/datasource/user/datasource_artifactory_user.go

```diff
@@ -0,0 +1,85 @@
+package user
+
+import (
+	"context"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/jfrog/terraform-provider-artifactory/v6/pkg/artifactory/resource/user"
+	"github.com/jfrog/terraform-provider-shared/util"
+	"github.com/jfrog/terraform-provider-shared/validator"
+)
+
+var userSchema = map[string]*schema.Schema{
```

Comment:
Let's move this variable into `DataSourceArtifactoryUser()` so it's scoped within that function.